src/modules/x509/extensions.ts

```diff
@@ -20,6 +21,12 @@ function textDecoder(rawExtension: Extension): string {
 	return UTF_8_DECODER.decode(rawExtension.value);
 }
 
+function utf8StringDecoder(rawExtension: Extension): string {
+	return AsnUtf8StringConverter.fromASN(
+		AsnAnyConverter.toASN(rawExtension.value)
+	);
+}
```

Comment:
@feelepxyz this snippet may be useful for the other places you are parsing these new extensions